redirect.json

```diff
@@ -151,19 +157,21 @@
   {
     "redirect": "lovelace_dashboards",
     "name": "Lovelace: Dashboards",
+    "badge": "Lovelace",
```

Comment:
This looks like the lovelace panel, and not dashboard config?

Reply:
hmm true. Make it Lovelace Config maybe? Dashboard is ambiguous too.

Reply:
```suggestion
    "badge": "Lovelace Config",
```